Makefile

```diff
@@ -37,6 +44,28 @@ $(DIRS):
 	@echo "⇒ Ensure dir: $@"
 	@mkdir -p $@
 
+# Directory for release files
+$(RELEASE):
+	@echo "⇒ Ensure dir: $@"
+	@mkdir -p $@
+
+# Prepare binaries and archives for release
+prepare-$(RELEASE): $(RELEASE) $(RELEASE_BINS)
+
+$(RELEASE)/%: dep
```

Comment:
This looks the same way as regular `dep`, apart from arch support. I suggest adding different arch support as a separate change when we need it. Currently, we support only amd64.

Reply:
why can't it be like already done? do not see any problems with that

at least it is different because of different targets that do not intersect(`BINS` and `RELEASE_BINS`)